src/Controller/CreatePayPalOrderFromCartAction.php

```diff
@@ -66,19 +71,33 @@ public function __construct(
                 ),
             );
         }
+        if (null === $this->orderPaymentsRemover) {
+            trigger_deprecation(
+                'sylius/paypal-plugin',
+                '1.6',
+                'Not passing an $orderPaymentsRemover to %s constructor is deprecated and will be prohibited in 2.0',
```

Comment:
```suggestion
                'Not passing an $orderPaymentsRemover to %s constructor is deprecated and will be prohibited in 3.0',
```
as 2.0 is already released and there will be the same backward compatibility problem